media/css/firefox/nothing-personal/_header.scss

```diff
@@ -0,0 +1,73 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+.c-page-header {
+    background-color: $browser-background;
+    border-bottom: $border-black;
+    position: sticky;
```

Comment:
**question:** I'm not sure what the best approach is here, but should this be static when prefers-reduced-motion is set, vs sticky?